soepy/shared/shared_constants.py

```diff
@@ -4,6 +4,7 @@
 MISSING_INT = -99
 INVALID_FLOAT = -99.0
 NUM_CHOICES = 3
+IND = 2  # Stands for indicator, i.e., an indicator variable takes 2 values, 0 or 1
```

Comment:
Is this really helpful to do? If so, please keep it otherwise drop it, please.

Reply:
Usage here was constructed identical to num choices. I just have a 2 now instead. After the speed refactor the loop regarding partner will disappear anyway.